Sources/DeepLearning/Operators/README.md

```diff
@@ -0,0 +1,41 @@
+# Ops and Convenience Methods
+
+The majority of the Tensor API is implemented in terms of 'ops' that are
+partitioned out to the TensorFlow graph when the compiler runs. These
+ops are intentionally designed to reflect TensorFlow ops, but provide nicer
+Swift syntax for accessing them. In addition to the core ops themselves,
+we also define some helper function wrappers, e.g. to make things symmetric
+and generally feel nice to use.
+
+The ops themselves are defined by the primitive `#tfop(...)` syntax, here 
```

Comment:
Most contents in this README file are actually out of date, since the compiler doesn't do GPE, there's no use of `#tfop`, and the `vjp:` label in `@differentiable` is considered an internal interface. The public interface is `@differentiating` specified on VJP functions. I think removing this file makes most sense. WDYT?

Reply:
Sounds good! I actually wasn't sure what the status is with respect to some of these comments and just wanted to make sure I don't delete them before you take a look. Btw, I think there's also some random TODOs throughout the code that may be outdated, but I haven't removed any of them for now.

Reply:
Thanks a lot for keeping things around!